components/raftstore/src/store/worker/split_check.rs

```diff
@@ -194,7 +194,7 @@ impl<S: CasualRouter<RocksEngine>> Runner<S> {
         let mut host = self.coprocessor.new_split_checker_host(
             &self.cfg,
             region,
-            &self.engine,
+            &self.engine.c(),
```

Comment:
Since this method will never be called in user request, we could create one object by `RocksEngine::from_db` and it could also be used in L216

Reply:
It's not necessary. Firstly because this is a free cast, not even an atomic clone, but also because this cast will go away in another patch very soon.